src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidator.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.RemoveDto;
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationMessages;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationUtils.getQualifiedFieldName;
+
+
+@Component
+public class RemoveValidator {
+
+    public Errors validate(RemoveDto removeDto, Errors errors, String loggingContext) {
+        if (removeDto.getIsNotProprietorOfLand() != null && !removeDto.getIsNotProprietorOfLand()){
```

Comment:
you can simply the if by using this instead - 
`if (Boolean.FALSE.equals(removeDto.getIsNotProprietorOfLand())) {
`

Reply:
statement redone